types/webxr/index.d.ts

```diff
@@ -266,6 +266,26 @@ interface XRInputSource {
 
 declare abstract class XRInputSource implements XRInputSource {}
 
+/**
+ * This Gamepad API interface represents hardware in the controller designed to provide haptic feedback to the user (if available), most commonly vibration hardware.
+ *
+ * [MDN Reference](https://developer.mozilla.org/docs/Web/API/GamepadHapticActuator)
+ */
+interface GamepadHapticActuator {
+    /**
+     * The pulse() method of the GamepadHapticActuator interface makes the hardware pulse at a certain intensity for a specified duration.
+     * @remarks Repeated calls to pulse() override the previous calls if they are still ongoing.
+     * @param value A double representing the intensity of the pulse. This can vary depending on the hardware type, but generally takes a value between 0.0 (no intensity) and 1.0 (full intensity).
+     * @param duration A double representing the duration of the pulse, in milliseconds.
+     * @returns A promise that resolves with a value of true when the pulse has successfully completed.
+     */
+    pulse(value: number, duration: number): Promise<boolean>;
+}
+
+interface Gamepad {
+    readonly hapticActuators: readonly GamepadHapticActuator[];
```

Comment:
This should probably be optional? This is not supported by all implementations. And the initial value (per specs) is undefined

Reply:
The reason I added it to this type is that, while the `hapticActuators property` is still experimental, it is often used with VR controllers. I always have to add some extra type information to avoid errors in my code. Other people in the community have mentioned this as well. 

The definition of the gamepad API itself is not a part of DT.  Do you think it's better to add a separate type for the Gamepad API to DT to incorporate this?